cwl/workflows/parts/1_mgygs/1.cwl

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
+class: Workflow
+
+requirements:
+  SubworkflowFeatureRequirement: {}
+  MultipleInputFeatureRequirement: {}
+  InlineJavascriptRequirement: {}
+  StepInputExpressionRequirement: {}
+  ScatterFeatureRequirement: {}
+
+inputs:
+  genomes_ena: Directory?
+  ena_csv: File?
+  genomes_ncbi: Directory?
+
+  max_accession_mgyg: int
+  min_accession_mgyg: int
+
+  # skip dRep step if MAGs were already dereplicated
+  skip_drep_step: string   # set "skip" for skipping
```

Comment:
Bools instead of strings should help simplify the code. We talked about this, but just to keep a note here :)

Reply:
Those parts files are only for testing. in final pipeline release I will get rid of this folder at all.
But I will change in all other used subwfs!